en/developer_guide/block_parameter.rst

```diff
@@ -0,0 +1,115 @@
+Block parameter
+===============
+
+Context
+-------
+
+To be properly displayed on the website, some block will require environment parameters.
+Those parameters could be of multiple types, for instance :
+
+ - url parameters
+ - server parameters
+ - header parameters
+ - ...
+
+An easy way to give those parameters to the block would have been to give all the request
+parameters when the block is called.
+
+This easy solution is not optimal for a main reason: when loading blocks via `ESI`_, each
+block will be cached for each url variant.
+
+If the block 1 does not require any parameters, in the following exemple :
+
+ * ``/block/1`` will be in one cache entry.
+ * ``/block/1?foo=bar`` will be in another cache entry.
+
+Open Orchestra solution
+-----------------------
+
+Open Orchestra gives you the opportunity to pass only the parameters you want
+to request a block, this way limits the cache entries for a single block.
+
+In the Backoffice, there is a part link to the block you add. In the Frontoffice, there
+is a part linked to the name of the parameter you will require.
```

Comment:
I still don't understand what the doc is talking about here :s
It says that Open Orchestra provides a mechanism to avoid multiple cache entries on Front Office, then it talks about stuff in Back Office. What are those "parts" ?